apps/src/dance/danceRedux.ts

```diff
@@ -0,0 +1,42 @@
+import {createSlice, PayloadAction} from '@reduxjs/toolkit';
+import {registerReducers} from '@cdo/apps/redux';
+
+type SongData = {
```

Comment:
Nice! Could we move this into a `types.ts` file? I'm guessing we'll want to reference this elsewhere eventually

Reply:
Love TS! Was wondering where these came from and see these keys in `songs.js`. Appreciate that they're declared as a type.

Reply:
Added!